src/Dlp/V2beta1/Gapic/DlpServiceGapicClient.php

```diff
@@ -20,12 +20,6 @@
  * This file was generated from the file
  * https://github.com/google/googleapis/blob/master/google/privacy/dlp/v2beta1/dlp.proto
  * and updates to that file get reflected here through a refresh process.
- *
- * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
- * more frequently than those which have been declared beta or 1.0, including changes which break
- * backwards compatibility.
- *
- * @experimental
```

Comment:
I think we need to remove the notice around the client not being beta for video intelligence as well.